protocol/0062-SPAM-spam_protection.md

```diff
@@ -63,7 +63,7 @@ The [on-chain referral program](./0083-RFPR-on_chain_referral_program.md) adds t
 - `UpdateReferralSet`
 - `ApplyReferralCode`
 
-To avoid spamming of `CreateReferralSet` and `UpdateReferralSet` transactions, a party must meet the staked governance tokens ($VEGA) threshold set by the network parameter `referralProgram.minStakedVegaTokens`. A party who does not meet this requirement should have any transactions of the aforementioned types pre-block rejected.
+To avoid spamming of `CreateReferralSet` and `UpdateReferralSet` transactions, a party must meet the staked governance tokens ($VEGA) threshold set by the network parameter `referralProgram.minStakedVegaTokens`. A party also must meet the balance holding threshold set by the network parameter `referralProgram.minBalance`. A party who does not meet this requirements should have any transactions of the aforementioned types pre-block rejected.
```

Comment:
do you mind calling the net param spam.protection.referralProgram.min.funds? to be consistent with spam.protection.applyReferral.min.funds?

Reply:
@Jiajia-Cui 

As this is a quantum value can this be made clear in the spec similar to the party profile spam:
https://github.com/vegaprotocol/specs/blob/2c7d4c5932bb03a257489b13f252256dde8afa44/protocol/0088-PPRF-party_profile.md?plain=1#L10